nominal/core/asset.py

```diff
@@ -215,6 +228,10 @@ def list_logsets(self) -> Sequence[tuple[str, LogSet]]:
         """
         return self._list_logsets()
 
+    @deprecated(
+        "LogSets are deprecated and will be removed in a future version. "
+        "Add logs to an existing dataset with dataset.write_logs instead."
+    )
```

Comment:
```suggestion
```

this is an internal method that's called by the method that's already got a deprecation warning